test/unit/liquid/drops/account_drop_test.rb

```diff
@@ -21,10 +21,17 @@ def test_url_helpers
     assert_equal @drop.id, @buyer.id
   end
 
+  test 'returns nil vat_rate for drop if buyer vate rate is nil' do
+    assert_nil @drop.vat_rate
+  end
+
   test 'returns vat_rate' do
-    assert_equal(@drop.vat_rate, @buyer.vat_rate)
+    @buyer.update(vat_rate: '123.4')
+    @drop_account = Drops::Account.new(@buyer)
+    assert_equal(@drop_account.vat_rate, @buyer.vat_rate)
```

Comment:
looks good, just a nit pick
```suggestion
    assert_equal @buyer.vat_rate, @drop_account.vat_rate
```